source/_integrations/braviatv.markdown

```diff
@@ -14,11 +14,30 @@ ha_config_flow: true
 
 The `braviatv` platform allows you to control a [Sony Bravia TV](https://www.sony.com/).
 
-Almost all [Sony Bravia TV 2013 and newer](https://info.tvsideview.sony.net/en_ww/home_device.html#bravia) are supported. A more generic method for older TVs connected to a Raspberry Pi with HDMI-CEC is explained further below.
+Almost all [Sony Bravia TV 2013 and newer](https://info.tvsideview.sony.net/en_ww/home_device.html#bravia) are supported. A more generic method for older TVs connected to a Raspberry Pi with HDMI-CEC is explained further [below](#For-TVs-older-than-2013).
 
+# Configuration
+## Setup via the User Interface
 You can setup the Sony Bravia TV via **Configuration** >> **Integrations** in the UI, click the button with `+` sign and from the list of integrations select Sony Bravia TV.
 
-If you have following configuration in your `configuration.yaml` file it will be imported to the Integrations:
+### Common Issues
+#### Previous Configurations:
+- Ensure that all entries in `configuration.yaml` have been removed and `bravia.conf` does not exist in your `.homeassistant` folder.
+- Remove entity from Home Assistant.
+- Restart Home Assistant
+- Retry [configuration](###Setup-via-the-User-Interface).
+
+#### TV does not generate new pin:
+- If you have previously set up your TV with any Home Assistant instances, you will need to remove Home Assistant from your TV. You can either factory reset your TV or follow [these instructions](https://www.sony-asia.com/electronics/support/articles/00129859). Your exact instructions may vary depending on yourmodel.
```

Comment:
Maybe you can add here something like this:
```
On the TV, go to: **Settings** -> **Network** -> **Remote device settings** -> **Deregister remote device** (menu item names may differ slightly for different TV models).
```
Maybe you can add something like this:
``
On the TV, go to: ** Settings ** -> ** Network ** -> ** Remote device settings ** -> ** Unregister remote device ** (menu item names may vary slightly depending on the TV model).
``
I checked several different TV models (2019, 2018 and 2017), and the names of the menu items were the same but in older models there are probably differences. Then factory reset may be necessary.

Reply:
I agree. pushing a commit to adjust this now.